pkg/gds/server.go

```diff
@@ -406,8 +478,7 @@ func (s *Server) Verification(ctx context.Context, in *api.VerificationRequest)
 
 // VerifyEmail checks the contact tokens for the specified VASP and registers the
 // contact email verification. If successful, this method then sends the verification
-// request to the TRISA Admins for review and generates a PKCS12 password in the RPC
-// response to decrypt the certificate private keys when they're emailed.
+// request to the TRISA Admins for review.
```

Comment:
Thank you for being so thorough with the comments and making sure the documentation is up to date! Great catch!